k8s/migratev2/templates/staging-claim.yaml

```diff
@@ -4,8 +4,6 @@ metadata:
   name: "{{.Release.Name}}-staging-claim"
 spec:
   storageClassName: {{ .Values.storageClassName }}
-  persistentVolumeReclaimPolicy: Delete
-  finalizers: []
```

Comment:
yep that was a hack to force the volume to be relinquished. Was never quite successful with that (k8s bug, might be solved now)

Reply:
```
Error: unable to build kubernetes objects from release manifest: error validating "": error validating data: [ValidationError(PersistentVolumeClaim.spec): unknown field "finalizers" in io.k8s.api.core.v1.PersistentVolumeClaimSpec, ValidationError(PersistentVolumeClaim.spec): unknown field "persistentVolumeReclaimPolicy" in io.k8s.api.core.v1.PersistentVolumeClaimSpec]
```